src/shared/components/form/form.css

```diff
@@ -1,6 +1,10 @@
 .form {
   display:flex;
   flex-direction: column;
+  width: 300px;
+  max-width: 100%;
+  margin: 0 auto 2rem;
+  height: 20rem;
```

Comment:
The static height (20 rem) applied to the form component is preventing all of the form fields from being displayed on the 'Create New Squad' page: 
![screenclip](https://cloud.githubusercontent.com/assets/8835055/26532835/fdcf6616-43d9-11e7-9d66-eb72e32f16f9.png)

Is there a reason to force all forms to have a height of 20rem? Forms can have a dynamic number of fields/rows (and therefore, height), so I'm learning towards removing this rule.